crates/bevy_render/src/render_phase/draw.rs

```diff
@@ -15,20 +15,23 @@ use bevy_ecs::{
 };
 use bevy_utils::HashMap;
 use parking_lot::{RwLock, RwLockReadGuard, RwLockWriteGuard};
-use std::{any::TypeId, fmt::Debug, hash::Hash, ops::Range};
+use std::{any::TypeId, fmt::Debug, hash::Hash};
 
-/// A draw function which is used to draw a specific [`PhaseItem`].
+// Todo: consider renaming this to `DrawFunction`
+/// A draw function used to draw [`PhaseItem`]s.
 ///
-/// They are the general form of drawing items, whereas [`RenderCommands`](RenderCommand)
-/// are more modular.
+/// Therefore the draw function can retrieve and query the required ECS data from the render world.
```

Comment:
```suggestion
/// The draw function can retrieve and query the required ECS data from the render world.
```

Not a fan of a use of "therefore" without a prior clause to refer back to.